src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+    // commit i push wysłać linka zadanie3 do git huba
+    public void gameProgress() {
+        int i = 0;
+        while (playerList.size() > 1) {
+            if (firstHand) {
+            } else {
+                try {
+                    Thread.sleep(4000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+            for (int n = 0; n < 20; n++) {
+                System.out.println();
+            }
+            System.out.println(player1.getStackPlayer());
+            System.out.println(player2.getStackPlayer());
+            if (i % 2 == 0) {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player1);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player2);
+                System.out.println("Zaczyna " + player2.getNamePlayer());
+                preFlopSmallBlind(player1, player2);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+            } else {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player2);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player1);
+                System.out.println("Zaczyna " + player1.getNamePlayer());
+                preFlopSmallBlind(player2, player1);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+            }
+            firstHand = false;
+            i++;
+        }
+    }
+
+    public void updatePlayers(String player) {
+        if (player.equals("Player1")) {
+            player1.increaseStackPlyer(gameState.pot);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
+        } else if (player.equals("Player2")) {
+            player2.increaseStackPlyer(gameState.pot);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
+        } else {
+            player1.increaseStackPlyer(gameState.pot / 2);
+            player2.increaseStackPlyer(gameState.pot / 2);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
```

Comment:
to do wyniesienia do nowej funckji updateStacksAndAmountOnShowdownResult(p1, p2, boolean isDraw, GameState gameState){
            p1.increaseStackPlyer(gameState.pot);
            p1.decreaseAmountInHand(gameState.pot / 2);
            p2.decreaseAmountInHand(gameState.pot / 2);
            gameState.decreasePot(gameState.pot);
            if(isDraw){
                 p2.increaseStackPlyer(gameState.pot / 2);
            }
}